src/odemis/acq/test/move_test.py

```diff
@@ -710,6 +897,55 @@ def test_lin_rot_axes(self):
         act_dist = _getDistance(point2, point1)
         self.assertAlmostEqual(exp_dist, act_dist)
 
+    def test_get_progress(self):
+        """
+        Test getMovementProgress function behaves as expected
+        """
+        start_point = {'x': 0, 'y': 0, 'z': 0}
+        end_point = {'x': 2, 'y': 2, 'z': 2}
+        current_point = {'x': 1, 'y': 1, 'z': 1}
+        progress = getMovementProgress(current_point, start_point, end_point)
+        self.assertTrue(util.almost_equal(progress, 0.5, rtol=RTOL_PROGRESS))
+
+        current_point = {'x': .998, 'y': .999, 'z': .999}  # slightly off the line
+        progress = getMovementProgress(current_point, start_point, end_point)
+        self.assertTrue(util.almost_equal(progress, 0.5, rtol=RTOL_PROGRESS))
+
+        # Same in the other direction
+        act_rot_error = _getDistance(point1, point2)
+        self.assertAlmostEqual(exp_rot_error, act_rot_error)
+
+    def test_rotation_axes_missing_axis(self):
+        point1 = {'rx': numpy.radians(30), 'rm': numpy.radians(30)}  # 30 degree
+        # No rx => doesn't count it
+        point2 = {'rm': numpy.radians(60)}  # 60 degree
+        exp_rot_dist = ROT_DIST_SCALING_FACTOR * numpy.radians(30)
+        act_rot_dist = _getDistance(point2, point1)
+        self.assertAlmostEqual(exp_rot_dist, act_rot_dist)
+
+        # Same in the other direction
+        act_rot_dist = _getDistance(point2, point1)
```

Comment:
Could you point out how the same values passed to _getDistance will verify 'other' direction?